tox.ini

```diff
@@ -1,25 +1,35 @@
 [tox]
-envlist = py33, py34, py35, flake8
+envlist = py27, py33, py34, py35, flake8
 
 [travis]
 python =
     3.5: py35
     3.4: py34
     3.3: py33
+    2.7: py27
 
 [testenv:flake8]
 basepython=python
 deps=flake8
 commands=flake8 partridge
 
 [testenv]
+basepython =
+    py27: python2.7
+    py33: python3.3
+    py34: python3.4
+    py35: python3.5
+    py36: python3.6
 setenv =
     PYTHONPATH = {toxinidir}
 deps =
-    -r{toxinidir}/requirements_dev.txt
+    -r{toxinidir}/requirements_dev_3x.txt
```

Comment:
Instead of doing this and the section below, just add a line after this like so:

```
deps =
    -r{toxinidir}/requirements_dev.txt
    py27: functools32
```

Reply:
Then we don't need two requirements files.